test/colony-funding.js

```diff
@@ -804,7 +807,7 @@ contract("Colony Funding", addresses => {
         colony.claimRewardPayout(payoutId2, initialSquareRoots, userReputation.toString(), totalReputation.toString(), {
           from: userAddress1
         }),
-        "colony-reward-payout-bad-id"
+        "colony-reward-payout-invalid-parameter-amount"
```

Comment:
The correct error message here should be `colony-token-locking-invalid-lock-id`. It says that there is invalid amount is because we are creating reward payout with colony's native token, which is not possible. Sorry, it was my mistake.
I'm not sure if i have permission to push a code to your branch so here is the correct test:
```javascript
const tokenArgs = getTokenArgs();
const newToken = await Token.new(...tokenArgs);
await fundColonyWithTokens(colony, newToken, initialFunding.toString());

await colony.startNextRewardPayout(otherToken.address);

const { logs } = await colony.startNextRewardPayout(newToken.address);
const payoutId2 = logs[0].args.id;

await checkErrorRevert(
  colony.claimRewardPayout(payoutId2, initialSquareRoots, userReputation.toString(), totalReputation.toString(), {
    from: userAddress1
  }),
  "colony-token-locking-invalid-lock-id"
);
```

Reply:
Thanks @filiplazovic. I'll change it to the correct test.